app/modules/booking/bookTickets/bookTicket.controller.js

```diff
@@ -1,21 +1,33 @@
 const { bookTicketQuery, decrementAvailableTickets } = require('./bookTicket.query');
 const { validationResult } = require('express-validator');
 const { sequelize } = require('../../../../models');
-
+const HttpStatus = require('http-status-codes');
+const auth = require('../bookTickets/authhelper');
 const bookTicket = async(req, res) => {
     try {
         let validation = validationResult(req);
         if (!validation.isEmpty()) {
-            return res.send(validation);
+            return res
+                .send(validation)
+        }
+
+        await auth.verifytoken(req, res);
+        console.log(req.user.id)
```

Comment:
Need to cross check unwanted comments and console.log through out the application before placing a pull request